apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -33,15 +32,20 @@ const AddressSource = new GQLListSchema('AddressSource', {
             isRequired: true,
             isUnique: true,
             hooks: {
+                resolveInput: ({ resolvedData, fieldPath }) => {
+                    if (has(resolvedData, fieldPath)) {
+                        return get(resolvedData, fieldPath, '').toLowerCase()
+                    }
+
+                    return get(resolvedData, fieldPath)
+                },
                 validateInput: async ({ resolvedData, fieldPath, context, existingItem, operation }) => {
-                    const value = resolvedData[fieldPath]
+                    const source = resolvedData[fieldPath]
```

Comment:
Maybe, you need to lowercase source here too?

Reply:
Good comment! Fixed :)